libsodium-sys/build.rs

```diff
@@ -22,16 +22,16 @@ use std::io::Cursor;
 use std::path::Path;
 
 static DOWNLOAD_BASE_URL: &'static str = "https://download.libsodium.org/libsodium/releases/";
```

Comment:
I wonder whether we should change it to github releases link, because download.libsodium.org is down sometimes